test/Storage/ImapTest.php

```diff
@@ -513,6 +513,27 @@ public function testAppend()
         $mail->appendMessage('');
     }
 
+    public function testAppendWithDate()
+    {
+        $mail = new Storage\Imap($this->params);
+        $count = $mail->countMessages();
+        $date = date('d-M-Y H:i:s O', strtotime('12/27/2015'));
+
+        $message = '';
```

Comment:
It definitely could be.  I did it that way to be consistent with previous tests (https://github.com/ameir/zend-mail/blob/05b0e63c8d5c2c534683473be32319da16df26f2/test/Storage/ImapTest.php#L504).  I'm not sure why it was that way, but I didn't want to break out of whatever standard was in place there, if any.